components/PageNav.vue

```diff
@@ -0,0 +1,115 @@
+<template>
```

Comment:
What about https://vuepress.vuejs.org/theme/default-theme-config.html#prev-next-links?

Reply:
I didn't understand your question. I haven't changed functionality there. Just moved from Page component to a new one.